src/main/java/core/basesyntax/service/BuyService.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service;
+
+import core.basesyntax.products.Fruit;
+import core.basesyntax.storage.ListStorage;
+import java.util.Comparator;
+import java.util.List;
+
+public class BuyService implements ServiceAble {
+
+    @Override
+    public void operationWithProduct(Fruit fruitToBuy) {
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names

Reply:
Changed interface name to Servicing, void method inside to seviceProduct(), used this
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html